src/styles/header-block.scss

```diff
@@ -0,0 +1,77 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-direction: row;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+
+  .logo {
+    display: flex;
+    justify-content: center;
+    height: 40px;
+    width: 40px;
+    margin: 10px 50px;
+  }
+
+  .nav {
```

Comment:
nav is a separate BEM-block:
![image](https://github.com/mate-academy/layout_catalog/assets/86923089/dcb8e695-f0b9-4c7d-ab31-d58adae6ca2c)

move it to a separate file